Src/Plugins/Tests.Security/TestData/ReviewPotentiallySensitiveData/Inputs/SEC102_003.Url.ps1

```diff
@@ -0,0 +1,4 @@
+https://msn.com/test
+https://msn.com/test/
+https://msn.com/test?foo=bar
+https://msn.com/test?foo=bar#bookmark
```

Comment:
In the tests, I cannot see this.

Reply:
that's by design, this is a bookmark, i.e., the fragment doesn't change the finding.